wrapanapi/virtualcenter.py

```diff
@@ -769,3 +769,65 @@ def usage_and_quota(self):
             'cpu_total': installed_cpu,
             'cpu_limit': None,
         }
+
+    def add_disk_to_vm(self, vm_name, capacity_in_mb, provision_type=None, unit=None):
+        """
+        Create a disk on the given datastore (by name)
+
+        Community Example used:
+        https://github.com/vmware/pyvmomi-community-samples/blob/master/samples/add_disk_to_vm.py
+
+        :param name: string name of the disk
+        :param capacity_in_mb: integer capacity, number of MegaBytes
+        :param type: string 'thin' or 'thick', defaults to thin
+        :param unit: unit number of disk, careful not to overwrite existing units
+        :return: ReconfigVM_Task object
```

Comment:
Noted, will use napoleon style.

Re-did the existing docblocks in vm_creation_time PR within this class: https://github.com/ManageIQ/wrapanapi/pull/151/commits/8a2bc25ec9fe25768548cd07fb87b58fe30310eb